### PR TITLE
Add default pre commit hooks

### PR DESCRIPTION
**Pull request by timoludwig**
### Short description

Add a few default pre-commit hooks, similar to https://github.com/digitalfabrik/integreat-compass/pull/27

Not sure whether the trailing new lines in the svg files are really necessary, but I guess for consistency it somehow makes sense...

__________________________________________________

[Pull Request Review Guidelines](https://digitalfabrik.github.io/integreat-cms/pull-request-review-guide.html)

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -1,4 +1,20 @@
...
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.4.0
+    hooks:
+      - id: check-added-large-files
+      - id: check-merge-conflict
+      - id: check-shebang-scripts-are-executable
+      - id: check-symlinks
+      - id: check-toml
+      - id: check-vcs-permalinks
+      - id: detect-private-key
```
Comment (charludo) [inline .pre-commit-config.yaml:11]: `detect-private-key` is a really good idea. Wondering if there's something similar / we could write something for API keys?

**Review by charludo — APPROVED** [review]
LGTM, and seems to be working nicely!

**Review by MizukiTemma — APPROVED** [review]
👍

File: .pre-commit-config.yaml (hunk 1 above)
Comment (timoludwig) [reply .pre-commit-config.yaml:11]: Good idea!
For this PR, this is probably out of scope, so I opened a new issue for this:
- https://github.com/digitalfabrik/integreat-cms/issues/2250

**Pull request merged**